src/apps/vguest/guest_vring.lua

```diff
@@ -0,0 +1,139 @@
+
```

Comment:
Module definition is missing:

```lua
module(...,package.seeall)
```
The module function is optional and it's mostly used because it's a good practice. It's widely used in SnS, so I think it's better to have it here too. In addition to that, this module is required from `vguest_dev.lua`.